contrib/babelfishpg_tsql/src/rolecmds.c

```diff
@@ -2243,6 +2243,38 @@ windows_login_contains_invalid_chars(char *input)
 	return false;
 }
 
+/**
+ * Domain name checks, doesnot allow characters like <>&*|' aong with list 
+ * https://github.com/MicrosoftDocs/SupportArticles-docs/blob/main/support/windows-se[…]rver/identity/naming-conventions-for-computer-domain-site-ou.md 
+*/
+bool
+windows_domain_contains_invalid_chars(char *input)
+{
+	int total_length = strlen(input);
+	char *pos_slash = strchr(input, '\\');
+	int	domain_len = total_length - strlen(pos_slash);
```

Comment:
we can simplify like this, `domain_len = pos_slash - input`.

Reply:
done